packages/client/utils/constants.ts

```diff
@@ -193,3 +193,25 @@ export const reasonsToDowngradeLookup: Record<
   "Not using Parabol's paid features": 'notUsingPaidFeatures',
   'Moving to another tool (please specify)': 'anotherTool'
 }
+
+export const timelineEventTypeMenuLabels: Record<string, string> = {
```

Comment:
+1 It would be nice to use `TimelineEventEnum` as key here. We could just import it from
```ts
import {TimelineEventEnum} from '../__generated__/MyDashboardTimelineQuery.graphql'
```
Type needs to be changed in `EventTypeFilterMenuItemLabel#Props` as well, but then it just works.